1223-curator_worklist/my_worklist_approval.feature

```diff
@@ -12,44 +12,37 @@ Feature: Curator handles an Approval Request
     As an Editor (at a medium Institution like SINTEF)
     I want to allow Users to publish metadata resources, then a Curator approv the resource before the content is made publicly accessable
 
-    Rules:
-     - The User got publisher privilege
-     - User got publisher privilege, but all content need a Curator Approval to be publicly accessable
-     - Only the Curator got publisher privilege
-
 	Background: 
-		Given a User that needs an approval to publish
+		Given a User wants to publish a Resource
+		And a Institutional publication policy that needs an approval to publish
 		And the User has created an Approval Request
 		And the Curator views the landing page of the Resource assosiated with the Request
 
 	@3188
 	Scenario: Curator approves an Approval Request to publish
-		Given the User wants to publish a Resource
-		And Institutional publication policy is "Only Curator can publish" 
+		Given Institutional publication policy is "Only Curator can publish" 
 		When the Curator approves the Request
 		Then the Request status is changed to "Approved"
 		And the Resource is published
 
 	@3191
 	Scenario: Curator declines an Approval Request
+		Given Institutional publication policy is "Only Curator can publish" 
 		When the Curator declines the Request 
-		And Institutional publication policy is "Only Curator can publish" 
 		Then the Request Status is set to "Declined"
 		And the Curator may send an answer Message with a reason to the User
 
 	@3189
 	Scenario: Curator approves an Approval Request to grant open access to content
-		Given the User has published a Resource
-		And Institutional publication policy is "Registrator can only publish metadata"
+		Given Institutional publication policy is "Registrator can only publish metadata"
 		When the Curator approves the Request
 		Then the Request status is changed to "Approved"
 		And the the Resources' content is open accessable 
-		# Don't remove the potentially original embargo on the files
+		And Don't remove the potentially original embargo on the files
```

Comment:
???

Reply:
Content may be locked and have an embargo.
Only remove the lock on the content, not the embargo.